File: app/src/main/java/com/wenitech/cashdaily/CustomerCreditFragment.java
package com.wenitech.cashdaily;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.navigation.Navigation;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Toast;

import com.wenitech.cashdaily.databinding.FragmentCustomerCreditBinding;

public class CustomerCreditFragment extends Fragment {

    private FragmentCustomerCreditBinding binding;

    public CustomerCreditFragment() {
        // Required empty public constructor
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        binding = FragmentCustomerCreditBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        CustomerCreditFragmentArgs args = CustomerCreditFragmentArgs.fromBundle(getArguments());
        if (args.getIsCreditActive()){
            binding.groupContentCustomer.setVisibility(View.VISIBLE);
            binding.groupCustomerVacio.setVisibility(View.GONE);
        }else {
            binding.groupContentCustomer.setVisibility(View.GONE);
            binding.groupCustomerVacio.setVisibility(View.VISIBLE);
        }

    }
}

File: app/src/main/java/com/wenitech/cashdaily/framework/features/client/customerCredit/ui/CustomerCreditFragment.kt
package com.wenitech.cashdaily.framework.features.client.customerCredit.ui

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.compose.material.ExperimentalMaterialApi
import androidx.compose.ui.platform.ComposeView
import androidx.fragment.app.Fragment
import androidx.fragment.app.viewModels
import androidx.navigation.NavController
import androidx.navigation.Navigation
import androidx.navigation.fragment.navArgs
import com.wenitech.cashdaily.R
import com.wenitech.cashdaily.databinding.FragmentCustomerCreditBinding
import com.wenitech.cashdaily.framework.composable.CustomerCreditScreen
import com.wenitech.cashdaily.framework.features.client.customerCredit.viewModel.CustomerCreditViewModel
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class CustomerCreditFragment : Fragment() {

    private val args by navArgs<CustomerCreditFragmentArgs>()
    private val viewModel by viewModels<CustomerCreditViewModel>()
    private lateinit var navController: NavController
    private lateinit var _binding: FragmentCustomerCreditBinding
    private val binding get() = _binding

    @ExperimentalMaterialApi
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?,
    ): View {
        // Inflate the layout for this fragment
        _binding = FragmentCustomerCreditBinding.inflate(inflater, container, false)
        return binding.root.apply {
            findViewById<ComposeView>(R.id.composeViewCustomerCredit).setContent {
                val idClient = args.idClient
                val refCredit = args.refCredit
                viewModel.setArgs(idClient, refCredit)

                CustomerCreditScreen(viewModel)
            }
        }
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        navController = Navigation.findNavController(view)
    }

}